tests/dwarf_common.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "data/dwarf_data.h"
+#include <fcntl.h>
+
+class test_dwarf : public ::testing::Test
+{
+protected:
+  void SetUp() override
+  {
+    char *bin = strdup("/tmp/dwarf_dataXXXXXX");
+    int fd = mkstemp(bin);
+    if (fd < 0)
+    {
+      std::remove(bin);
```

Comment:
is this correct?

Reply:
It's not, `mkstemp` doesn't create a file on failure...I just followed what is used in `tests/btf_common.h`.